app/postinstallscript.sh

```diff
@@ -27,9 +29,14 @@ Wants=acap-user@$_uid.service" >> /etc/systemd/system/sdkdockerdwrapper.service
 
 fi
 
-# Create mapping for subuid and subgid - both shall use user name!
-echo "$_uname:100000:65536" > /etc/subuid
-echo "$_uname:100000:65536" > /etc/subgid
+# Create mapping for subuid and subgid - both shall use user id!
```

Comment:
Is there an easily understood motivation, so you can write `both shall use user id, because ...`?

Reply:
Yes, but too long for a temporary shell script. This is more of a reminder that uid should be used even in /etc/subgid